plugins/config/docker-args-deploy

```diff
@@ -3,8 +3,9 @@ set -eo pipefail; [[ $DOKKU_TRACE ]] && set -x
 source "$PLUGIN_PATH/common/functions"
 source "$PLUGIN_PATH/config/functions"
 
-STDIN=$(cat); APP="$1"; IMAGE="dokku/$APP"
+STDIN=$(cat); APP="$1"; TAG="$2"
 DOCKERFILE_ENV_FILE="$DOKKU_ROOT/$APP/DOCKERFILE_ENV_FILE"
+verify_app_name "$APP" "$TAG"
 
 if ! is_image_herokuish_based "$IMAGE"; then
```

Comment:
Yeah this requires a central place for setting `$IMAGE`. I suppose we can break it up into another call.

something like
```
STDIN=$(cat); APP="$1"; TAG="$2"
verify_app_name "$APP"
IMAGE=$(get_app_image "$APP" "$TAG")
```